tvbwidgets/core/pse/parameters.py

```diff
@@ -0,0 +1,340 @@
+"""
```

Comment:
Missing header. Pls copy from rest of modules.
Ideally also mention Jan Fousek as author, and the fact that this is a temporary copy from tvb-inversion package